src/commands/force/org/list.ts

```diff
@@ -159,31 +168,40 @@ export class OrgListCommand extends SfdxCommand {
     }
   }
 
-  private getScratchOrgColumnData(): unknown[] {
+  private getScratchOrgColumnData(): Partial<CliUx.Table.table.Columns<Record<string, string>>> {
```

Comment:
I remember setting the type for column data to `unknown` because it was giving me conflicts with the new table types, I'm glad you solved that, but I get the sense that the type is overly complex for just a simple column data descriptor, I'm sure It won't hurt us in the short~mid term but breaking changes in the library could happen and this is where it gets messy.

Reply:
yep, the whole import looks complex but that's actually just cli-ux after being merge into oclif/core, here's the previous type: https://github.com/salesforcecli/plugin-org/blob/15ae104f9d6be2d1ec1678424db67548312a2ac5/src/commands/force/org/list.ts#L157